### ISSUE TITLE
Score method??

### ISSUE THREAD

**Issue by alexpghayes** [alexpghayes/modelling-in-r#20]
Scikit-Learn provides the `score` method to quickly assess the performance of a model on new data.

Is this a worthwhile thing to provide or too much from Python land and foreign?

Could extract a performance metric from the `calibration` object and use that to get consistent behavior.

**Comment by alexpghayes** [comment]
In retrospect, maybe this is `broom::glance`?

For a single model, it makes sense for `broom::glance` to return a single row of metrics. For a model family, would it be okay to return multiple rows, one for each set of hyperparameters?

**Comment by alexpghayes** [comment]
Moving to main text
